packages/snapshot-listener/src/index.ts

```diff
@@ -41,7 +41,7 @@ log.info(
 );
 
 const app = express();
-const port = process.env.PORT || DEFAULT_PORT;
+const port = process.env.SL_PORT || DEFAULT_PORT;
```

Comment:
Pretty sure this breaks the snapshot listener. The `PORT` environment variable is automatically set by Heroku on dyno start-up.